Sources/YapDB.swift

```diff
@@ -405,20 +411,20 @@ extension YapDB {
     is similar to a full text search extension. It features a handler, which must
     be provided to update a dictionary used to index records.
     */
-    public class SecondaryIndex: BaseExtension, YapDatabaseExtensionRegistrar {
+    open class SecondaryIndex: BaseExtension, YapDatabaseExtensionRegistrar {
```

Comment:
I prefer to leave these classes `open` so consumers of this pod can subclass them if that is useful for their application.